README.md

```diff
@@ -47,8 +47,10 @@ Define a `current_user` in your `application_controller.rb`:
 
 ```ruby
 class ApplicationController < ActionController::Base
+    # ...
```

Comment:
Not sure why there's an ellipsis here?

Reply:
Oh, just to imply that there is more code there that I snipped out to abbreviate. I will remove it, it's probs a little confusing.